tests/foreman/cli/test_errata.py

```diff
@@ -419,12 +928,54 @@ def test_positive_list_filter_by_product_id_and_org_name(self):
         1. erratum list --product-id=<product_id> --organization=<org_name>
 
         @Assert: Errata is filtered by product id and Org name.
-
-        @caseautomation: notautomated
-
         """
-
-    @stubbed()
+        org_product_erratum_list = Erratum.list({
+            'organization': self.org['name'],
+            'product-id': self.org_product['id'],
+            'per-page': 1000
+        })
+        org_product_errata_ids = {
+            errata['errata-id']
+            for errata in org_product_erratum_list
+        }
+        org3_product1_erratum_list = Erratum.list({
+            'organization': self.org3['name'],
+            'product-id': self.org3_product1['id'],
+            'per-page': 1000
+        })
+        org3_product1_errata_ids = {
+            errata['errata-id']
+            for errata in org3_product1_erratum_list
+        }
+        org3_product2_erratum_list = Erratum.list({
+            'organization': self.org3['name'],
+            'product-id': self.org3_product2['id'],
+            'per-page': 1000
+        })
+        org3_product2_errata_ids = {
+            errata['errata-id']
+            for errata in org3_product2_erratum_list
+        }
+        self.assertEquals(len(org_product_errata_ids), 4)
+        self.assertEquals(len(org3_product1_errata_ids), 4)
+        self.assertEquals(len(org3_product2_errata_ids), 79)
```

Comment:
Imo better to introduce constants here - in case we decide to change repositories we have to change 1 line in setupclass... and every single test because errata counts don't match